postgresql/default.toml

```diff
@@ -1,7 +1,8 @@
 port = '5432'
 max_connections = 500
-log_line_prefix = "%m: "
+log_line_prefix = '%t [%p]: [%l-1] user=%u,db=%d,client=%h %r (%x:%e)'
 log_level = 'ERROR'
+data_checksums = true
```

Comment:
Good point actually, I think you are right and we just leave it on till there is a use case for off?